examples/delegated-routing/.env

```diff
@@ -0,0 +1,3 @@
+# required because react-scripts scans *up* the tree from this project and finds
+# a conflicting version of eslint in the node_modules dir for js-ipfs.
```

Comment:
Is this necessary?  js-ipfs doesn't have a runtime dependency on eslint?  Maybe if this example was in the js-ipfs repo?

Reply:
This is needed. I copied this from the react example of `js-ipfs`, but did not read the comment: https://github.com/ipfs/js-ipfs/blob/master/examples/browser-create-react-app/.env
It is aegir that installs a version of `webpack` that is not compatible with `react-scripts`. I am going to change that comment